### ISSUE TITLE
Unique viewers controls present on page analytics are not accessible with keyboard.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#17442]
**User Impact**
Unique viewers controls are not accessible by keyboard user depending only on keyboard will not be able to access these controls.

**Expected behavior**
Unique viewers controls present on page analytics should be accessible with keyboard also in scan mode also.

**Actual behavior**
Unique viewers controls present on page analytics dialog are not accessible with keyboard.
Observation-Unique viewers controls present on page analytics are not accessible in scan mode while navigating with narrator. 

**Test Environment:**
OS: Windows 10 21H2 (build 19044.1766)
Browser: Microsoft Edge Version 103.0.1264.37 (Official build) (64-bit)
Demo URL: https://codepen.io/Harithalakshmi/pen/yLvZyqR

**Steps to reproduce**

1-Open the URL in edge browser
2-Turn on narrator
3.Navigate to preview
4-Navigate to "Chart context menu"
5-After Chart context menu there are viewer controls (0.00-10.00 11.00-100.00 101.00-150.00)
6-These controls are not accessible by keyboard with narrator these are not accessible in scan off/on mode. 

Attachments

https://user-images.githubusercontent.com/93735775/175989438-1f1f4da7-f525-4369-a968-3dc6917a6476.mp4

**Comment by oysteinmoseng** [comment]
Internal note: Color axis with data classes is not kbd accessible. Minimal demo: https://jsfiddle.net/oysteinmoseng/k63z1sbw/

**Comment by ashish315** [comment]
@TorsteinHonsi As checked the issue still repro, can you please let me know the version in which the fix is deployed?
![image](https://user-images.githubusercontent.com/60913131/179714363-ef3ebc7f-bd3c-40c9-9fef-a56ac99e424e.png)

**Comment by raf18seb** [comment]
Hi @ashish315,
A fix for this bug was merged on July 7, two days after the v10.2.0 was released. It means that a fix will be available in the next version (10.2.1 or 10.3.0).

**Comment by ashish315** [comment]
Hi @raf18seb Is it possible to have fix ported in the older versions such as high chart version 10.2.0 or below

**Comment by karolkolodziej** [comment]
Hi @ashish315, unfortunately, this is not possible.
Also, I was not able to wrap up these changes into something reusable in the previous versions.
In order to use that functionality you have to update it to a version greater or equal to 10.2.1.

**Comment by ashish315** [comment]
Thanks for the info!
